capirca/aclgen.py

```diff
@@ -482,23 +495,46 @@ def Run(base_directory, definitions_directory, policy_file, output_directory,
     logging.info('rendering one file')
     RenderFile(base_directory, policy_file, output_directory, definitions,
                FLAGS.exp_info, write_files)
+  elif FLAGS.max_renderers == 1:
+    # If only one process, run it sequentially
+    policies = DiscoverAllPolicies(
+      base_directory,
+      output_directory,
+      definitions
+    )
+    result = []
+    for pol in policies:
+      RenderFile(
+        base_directory,
+        pol.get('in_file'),
+        pol.get('out_dir'),
+        definitions,
+        FLAGS.exp_info,
+        write_files
+      )
   else:
     # render all files in parallel
-    logging.info('finding policies...')
-    pols = []
-    pols.extend(DescendRecursively(base_directory, output_directory,
-                                   definitions))
-
+    policies = DiscoverAllPolicies(
+      base_directory,
+      output_directory,
+      definitions
+    )
     pool = context.Pool(processes=FLAGS.max_renderers)
     results = []
```

Comment:
Also seems unused (in original code)

Reply:
It's used to store all the promises, I think, but I'm not 100% sure why